modules/pipeline/pipengine/actionexecutor/logic/job.go

```diff
@@ -60,6 +61,24 @@ func ParseJobHostBindTemplate(hostPath string, clusterInfo apistructs.ClusterInf
 	return b.String(), nil
 }
 
+func GenerateDockerVolumes(job *apistructs.JobFromUser) []mount.Mount {
```

Comment:
I think we should consider why the pkg called `logic` and if it is suitable to put docker logic here?

Reply:
adjusted